src/main/java/com/company/M11/Main.java

```diff
@@ -0,0 +1,69 @@
+package com.company.M11;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.concurrent.atomic.AtomicInteger;
+import java.util.stream.IntStream;
+import java.util.stream.Stream;
+
+public class Main {
+    public static void main(String[] args) {
+//        task1();
+//        task2();
+//        task3();
+//        task4();
+//        task5();
+    }
+
+    private static void task1() {
+        List<String> names = Arrays.asList("John", "Bill", "Anthon", "Alex");
+
+        AtomicInteger index = new AtomicInteger(0);
+        Stream<String> namesStream = names.stream().filter(name -> index.getAndIncrement() % 2 != 0);
```

Comment:
please start a new operation with dot from new line when you working with Stream, this is a common approach. Fix here and in all places in the code
```suggestion
        Stream<String> namesStream = names.stream()
        .filter(name -> index.getAndIncrement() % 2 != 0);
```

Reply:
fixed